identity-access-manager-keycloak/package-metadata.json

```diff
@@ -6,7 +6,6 @@
   "version": "0.0.1",
   "dependencies": [],
   "environmentVariables": {
```

Comment:
I guess STATEFUL_NODE should be replaced by CLUSTER_MODE ?

Reply:
We agreed on removing this variable from package-metadata.json because it is not really an environment variable